### PR TITLE
[Ubuntu] Downgrade git-lfs version to 2.13.3

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
# Description
There is an issue with newest git-lfs version 3.0.0. We need to downgrade to 2.13.3 version until issue is fixed.
Successful run: https://buildcanary.visualstudio.com/PipelineCanary/_build/results?buildId=1811517&view=results

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by dmitry-shibanov — DISMISSED** [review]

**Review by MaksimZhukov — DISMISSED** [review]

**Review by al-cheb — DISMISSED** [review]

**Comment by catthehacker** [comment]
Maybe it would be better to use GitHub release instead of apt package? that way it will be one repo less to keep track of
https://github.com/git-lfs/git-lfs/releases/latest

File: images/linux/scripts/installers/git.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ git --version
 
 # Install git-lfs
 curl -s $GIT_LFS_REPO/script.deb.sh | bash
-apt-get install -y git-lfs
+apt-get install -y git-lfs=2.13.3 --allow-downgrades
```
Comment (miketimofeev) [inline images/linux/scripts/installers/git.sh:21]: Why do we need `--allow-downgrades` flag?

**Comment by miketimofeev** [comment]
> Maybe it would be better to use GitHub release instead of apt package? that way it will be one repo less to keep track of [git-lfs/git-lfs/releases/latest](https://github.com/git-lfs/git-lfs/releases/latest)

I can't recall why we decided to preserve the installation from ppa, but we will definitely revisit this, thanks!

File: images/linux/scripts/installers/git.sh (hunk 1 above)
Comment (dibir-magomedsaygitov) [reply images/linux/scripts/installers/git.sh]: Thank you, removed unnecessary flag

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**